applications/asset_tracker_v2/src/modules/location_module.c

```diff
@@ -248,6 +248,13 @@ static void search_start(void)
 		methods_count++;
 	}
 
+#if defined(CONFIG_LOCATION_METHOD_WIFI)
+	if (!copy_cfg.no_data.wifi) {
+		methods[methods_count] = LOCATION_METHOD_WIFI;
+		methods_count++;
+	}
+#endif
```

Comment:
Maybe this was discussed already but somehow was thinking that if WiFi is included in config, then we shall use that and fallback to GNSS/Cellular, i.e. in areas where wifi is not available. In that we way we could get quite accurate location quite fast also city areas and indoors.

Reply:
That's not possible with the current design of ATv2. If we add WiFi to the top of the list, we will always request GNSS as ATv2 doesn't wait for the response from the cloud on whether the location was resolved successfully or not but it tells that location result is unknown to Location library.

Reply:
OK, thanks for the clarification. For me that sounds like a possible future PR for ATv2.